File: problemas/iniciante/2708/2708.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.split('\n');

    let [pessoas, jipes] = [0, 0];
    let [comando, visitantes] = entrada.shift().trim().split(' ');

    while (comando !== "ABEND") {
        if (comando === "SALIDA") {
            pessoas += parseInt(visitantes);
            ++jipes;
        } else {
            pessoas -= parseInt(visitantes);
            --jipes;
        }

        [comando, visitantes] = entrada.shift().trim().split(' ');
    }   

    console.log(pessoas);
    console.log(jipes);
});

File: problemas/iniciante/2708/2708.py
def main():
    jipes = 0
    pessoas = 0
    while True:
        entrada  = input().split()

        Jipe = entrada[0]
        if Jipe == "ABEND":
            break

        Visitantes = int(entrada[1])

        if Jipe == "SALIDA":
            jipes += 1
            pessoas += Visitantes
        
        elif Jipe == "VUELTA":
            jipes -= 1
            pessoas -= Visitantes

    print(pessoas)
    print(jipes)
            
if __name__ == "__main__":
    main()
